app/controllers/access_tokens_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccessTokensController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    token_expiration_time = Rails.application.config.access_token_expiration_time
+    current_user.access_tokens.create!(
+      token: SecureRandom.hex(16),
+      expired_at: token_expiration_time.from_now
+    )
```

Comment:
この処理はAccessTokenモデルのメソッドに持って行けそうです。
つぎのようになると、良さそうです。

```suggestion
    current_user.access_tokens.create!
```

token_expiration_timeもトークンからとれた方が綺麗ですね。
メッセージの内容が期間でなくて、日時でも良いかもしれません。

```ruby
"It will expired at #{token.expired_at.to_s}."
```
みたいなイメージです。


もしかするとUserクラスのメソッドにして
```ruby
token = current_user.create_token!
```
にするのがよいかもしれません。

Reply:
ご提案ありがとうございます。
Userクラスのメソッドの方が個人的にはイメージが近かったので、そのように実装してみました。
合わせて日時も修正しました。
https://github.com/pubannotation/pubdictionaries/pull/142/commits/e6d7d4cf141b463650171c535c896b50c17653a5